swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -6623,6 +6725,13 @@ components:
           $ref: "#/components/schemas/fundsAvailable"
         psuMessage:
           $ref: "#/components/schemas/psuMessageText"
+        _links:
+          $ref: "#/components/schemas/_linksPaymentInitiationStatus"
```

Comment:
Relevant Change: _linksPaymentInitiationStatus is new
Adoption approach: Follow the change